internal/provisioner/kops_provisioner.go

```diff
@@ -210,8 +200,8 @@ func (provisioner *KopsProvisioner) k8sClient(clusterName string, logger log.Fie
 	var k8sClient *k8s.KubeClient
 	k8sClient, err = k8s.NewFromFile(configLocation, logger)
 	if err != nil {
-		return nil, nil, errors.Wrap(err, "failed to create k8s client from file")
+		return nil, errors.Wrap(err, "failed to create k8s client from file")
 	}
 
-	return k8sClient, invalidateOnError, nil
```

Comment:
Was `invalidateOnError` unnecessary?

Reply:
`invalidateOnError` was never used.  I mean It was called by defer but defer was set after the error check. So It was necessary. Thats why I have removed.